src/main/java/com/battleships/battleships/BattleshipSquare.java

```diff
@@ -1,13 +1,17 @@
 package com.battleships.battleships;
 
+//inheritance versus aggregation?
 public class BattleshipSquare {
 
     private String type;
+    //
     private String state;
     private int space;
     private int spaceCount;
     private String hitValue;
+    //
     private String hitMessage;
+    //
```

Comment:
`destroyedType` just feels weird especially given a default String value of `None`.